### PR TITLE
Docs 7912 fix edit comments and misc pa

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/bind-api-configure-service-mesh-CRD.adoc [hunk 1]
```diff
@@ -43,12 +43,16 @@ To bind the service with an API, perform the following steps:
...
+image::bind-api-id.png[Display API ID]
++
+<1> API name and API instance version.
+<2> API ID.
 +
 . Retrieve from Kubernetes the name of the service with which you want to bind the API:
 +
 `kubectl get svc -n <namespace>`
-. Verify that your microservice port is named following Istio standards: xref:https://istio.io/docs/setup/additional-setup/requirements/[].
+. Verify that your microservice port is named based on xref:https://istio.io/docs/setup/additional-setup/requirements/[Istio requirements].
```
Comment (fermujica) [inline modules/ROOT/pages/bind-api-configure-service-mesh-CRD.adoc]:
```suggestion
. Verify that your microservice port is named based on link:https://istio.io/docs/setup/additional-setup/requirements/[Istio requirements].
```

Comment (fermujica) [reply modules/ROOT/pages/bind-api-configure-service-mesh-CRD.adoc]: xref are only for internal references

File: modules/ROOT/pages/configure-service-mesh.adoc [hunk 2]
```diff
@@ -7,12 +7,12 @@ endif::[]
 image::configure-service-mesh-breadcrumb.png[]
 
 
-[[adapter-overview]]
-== Overview
+// [[adapter-overview]]
+// == Overview
```
Comment (fermujica) [inline modules/ROOT/pages/configure-service-mesh.adoc:11]: Why are you commeting out titles and anchors? Are you using them somewhere else? 
If these are not necessary, I'd suggest removing them.

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -3,5 +3,67 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Service Mesh Beta program was launched in October 2019. 
+:page-aliases: service-mesh-overview-and-landing-page.adoc
```
Comment (fermujica) [inline modules/ROOT/pages/index.adoc]: If this was never in production, there is no need for a page-alias

**Review by fermujica — APPROVED** [review]
Please check my comments before merging

File: modules/ROOT/pages/configure-service-mesh.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/configure-service-mesh.adoc:11]: I don't see them being used anywhere. I just want to check I once. I'll remove these the release tomorrow.

File: modules/ROOT/pages/bind-api-configure-service-mesh-CRD.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/bind-api-configure-service-mesh-CRD.adoc]: Got it! Thanks! Done!

File: modules/ROOT/pages/index.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/index.adoc]: Ok! Thanks! Done!

**Pull request merged**